internal/pkg/call/call.go

```diff
@@ -64,22 +71,31 @@ func ValidBackend(backendName string) bool {
 	return false
 }
 
-func CallBackend(ctx context.Context, callData *data.Call, leaveTmpFile, printCommand bool) (string, error) {
+func CallBackend(ctx context.Context, callData *data.Call, leaveTmpFile, printCommand bool) (Output, error) {
 	backendTimeoutContext := ctx
 	if callData.Config.Timeout != data.TimeoutNotSet {
 		backendTimeoutContext, _ = context.WithTimeout(ctx, time.Duration(callData.Config.Timeout)*time.Second)
 	}
 
 	backend, err := getBackend(callData)
 	if err != nil {
-		return "", errors.Wrapf(err, "Could not instantiate backend: %s", callData.Backend)
+		return Output{}, errors.Wrapf(err, "Could not instantiate backend: %s", callData.Backend)
 	}
 
 	if printCommand {
-		return backend.getAsString()
+		if command, err := backend.getAsString(); err != nil {
+			return Output{StdErr: []byte(command)}, err
+		} else {
+			return Output{StdOut: []byte(command)}, nil
```

Comment:
Change getAsString() into returning a []byte array with a matching method name, it's currently converting a string to a []byte and then into a string again